invenio_records_marc21/templates/invenio_records_marc21/records/details/subjects.html

```diff
@@ -0,0 +1,18 @@
+{# -*- coding: utf-8 -*-
+
+  Copyright (C) 2021 Graz University of Technology.
+
+  Invenio-Records-Marc21 is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+
+{% set subjects = metadata.get('subject_added_entry_corporate_name') %}
+{% if subjects %}
+<dt><b>{{_('Subjects')}}</b></dt>
```

Comment:
i know inveniosoftware is using the dt dd construction, but is this valid html code without a surrounding dl tag? as i remember, in the rdm-records template there is no surrounding dl tag.

Reply:
i am not adding those messages to all dl dd tags